x-pack/plugins/security_solution/public/common/components/charts/common.test.tsx

```diff
@@ -23,15 +23,6 @@ import { LEGACY_LIGHT_THEME, LEGACY_DARK_THEME } from '@elastic/charts';
 
 jest.mock('../../lib/kibana');
 
-jest.mock('@elastic/charts', () => {
-  const original = jest.requireActual('@elastic/charts');
-
-  return {
-    ...original,
-    getSpecId: jest.fn(() => {}),
-  };
-});
```

Comment:
This is now superfluous. Chart `ids` used to be cast as unique strings via the `getSpecId` helper (see [example](https://github.com/markov00/elastic-charts/blame/8afa62afc4206d8ce3876aa5ccd9c69a8c20518a/src/utils/ids.ts)) but this was changed in  https://github.com/elastic/elastic-charts/pull/281 and fully removed in https://github.com/elastic/elastic-charts/pull/554.